internal/server/bunextRequest.ts

```
@@ -48,36 +57,36 @@ export class BunextRequest {
       [this.path]: data,
     };
   }
-  public setCookie(response: Response) {
+  public async setCookie(response: Response) {
     switch (globalThis.serverConfig.session?.type) {
       case "database:hard":
       case "database:memory":
         const correctID = this.SessionID || generateRandomString(32);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)